src/components/victory-voronoi.jsx

```diff
@@ -71,26 +71,28 @@ class VictoryVoronoi extends React.Component {
 
       const path = "M" + vertices.join("L") + "Z";
 
-      return (
+      const _buildSvg = voronoi => {
```

Comment:
hmm, you should be able to break it out of the `data` prop like `data.path` maybe?  Eventually it will take the pattern we are using in the other components:
```
if (this.props.animate) {
      // Do less work by having `VictoryAnimation` tween only values that
      // make sense to tween. In the future, allow customization of animated
      // prop whitelist/blacklist?
      const animateData = _.omit(this.props, [
        "animate", "scale", "standalone", "interpolation"
      ]);
      return (
        <VictoryAnimation {...this.props.animate} data={animateData}>
          {props => <VLine {...this.props} {...props}/>}
        </VictoryAnimation>
      );
    }
    return (<VLine {...this.props}/>);
```